config/locales/en.yml

```diff
@@ -6,7 +6,107 @@ en:
       emails_found: 
         one: "one email found"
         other: "%{count} emails found"
-  
+
+  errors:
```

Comment:
I think this might be left over from the challenge assignment email. I think it'll create merge conflicts if we take out out, so it can stay, but let's make sure to merge #2773 first.